lib/premailer/rails/css_helper.rb

```diff
@@ -26,7 +26,7 @@ def css_for_url(url)
       private
 
       def css_urls_in_doc(doc)
-        doc.search('link[@rel="stylesheet"]').map do |link|
+        doc.search("link[@rel='stylesheet']:not([@data-premailer='ignore'])").map do |link|
```

Comment:
I'd prefer to stick to the single quote string and use double quote strings inside, i.e., flip the quotes :blush: